.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Never modify lines endings
```

Comment:
Can you add a note to this comment that we're disabling it because line normalization can break integration tests if `core.autocrlf=true`?